dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/DimensionDataCloudControlProviderMetadata.java

```diff
@@ -42,12 +47,26 @@ public DimensionDataCloudControlProviderMetadata() {
       super(builder());
    }
 
-   public DimensionDataCloudControlProviderMetadata(Builder builder) {
+   public DimensionDataCloudControlProviderMetadata(final Builder builder) {
       super(builder);
    }
 
    public static Properties defaultProperties() {
       Properties properties = DimensionDataCloudControlApiMetadata.defaultProperties();
+      properties.setProperty(PROPERTY_REGIONS, "na,eu,au,mea,ap,canada");
+      properties.setProperty(PROPERTY_ISO3166_CODES, "NA,EU,AU,AF,AP,CA");
+      properties.setProperty(PROPERTY_REGION + ".na." + ISO3166_CODES, "NA");
+      properties.setProperty(PROPERTY_REGION + ".na.zones", "NA9,NA12");
+      properties.setProperty(PROPERTY_REGION + ".eu." + ISO3166_CODES, "EU");
+      properties.setProperty(PROPERTY_REGION + ".eu.zones", "EU6,EU7,EU8");
+      properties.setProperty(PROPERTY_REGION + ".au." + ISO3166_CODES, "AU");
+      properties.setProperty(PROPERTY_REGION + ".au.zones", "AU9,AU10,AU11");
+      properties.setProperty(PROPERTY_REGION + ".mea." + ISO3166_CODES, "AF");
+      properties.setProperty(PROPERTY_REGION + ".mea.zones", "AF3");
+      properties.setProperty(PROPERTY_REGION + ".ap." + ISO3166_CODES, "AP");
+      properties.setProperty(PROPERTY_REGION + ".ap.zones", "AP4,AP5");
+      properties.setProperty(PROPERTY_REGION + ".canada." + ISO3166_CODES, "CA");
+      properties.setProperty(PROPERTY_REGION + ".canada.zones", "CA2");
```

Comment:
Consider fixing the ISO3166 code values and put the right ones for each physical region and zone: https://en.wikipedia.org/wiki/ISO_3166-2

You can configure ISO3166 codes for each zone by adding properties such as:
```java
properties.setProperty(PROPERTY_ZONE + "EU7" + ISO3166_CODES, "DE-BE");
```

Reply:
thanks @nacx I can try figure those out.